src/controllers/user/verify-email.ts

```diff
@@ -49,7 +52,12 @@ export const getVerifyEmailController = async (
       pageTitle: "Vérifier votre email",
       notifications: await getNotificationsFromRequest(req),
       email,
-      csrfToken: csrfToken(req),
+      countdownEndDate: moment(verify_email_sent_at)
+        .add(MIN_DURATION_BETWEEN_TWO_VERIFICATION_CODE_SENDING_IN_SECONDS, "s")
+        .tz("Europe/Paris")
+        .locale("fr")
+        .format(),
+      csrfToken: email && csrfToken(req),
```

Comment:
suggestion: the email test is not useful anymore

```suggestion
      csrfToken: csrfToken(req),
```